Makefile

```diff
@@ -0,0 +1,31 @@
+IMAGE_NAME_PUBGRADE=akash7778/pubgrade
+IMAGE_NAME_UPDATER=akash7778/pubgrade-updater
+APP_NAME=pubgrade
+
+
+
+.PHONY: help
+
+help: ## This help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.DEFAULT_GOAL := help
+
+
+build: ## Builds two container images (main-service, build-complete-updater).
+	docker build -t $(IMAGE_NAME_PUBGRADE) .
+	docker build -f build-complete-updater/Dockerfile -t $(IMAGE_NAME_UPDATER) .
```

Comment:
Is this error expected?

`unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/galvaro/src/Pubgrade/build-complete-updater: no such file or directory`

Am I doing something wrong?

Reply:
Fixed! Rebase was the problem